src/posts/2020-10-14-wcag2-documents.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Introduction to WCAG 2.0
+description: A quick overview of WCAG 2 documents.
+category: Background
+author: Hamsa Harcourt
+further_reading:
+  - url: https://www.w3.org/TR/WCAG20/#intro-layers-guidance 
+    title:  Web Content Accessibility Guidelines (WCAG) 2.0
+    source: W3C 
+  - url:  https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines
+    title: Web Content Accessibility Guidelines
+    source: Wikipedia
+date: 2020-10-14
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - background
+---
+
+
+The [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/) (<abbr>WCAG</abbr>) are web accessibility guidelines published by the **[Web Accessibility Initiative](https://www.w3.org/WAI/about/)** (<abbr>WAI</abbr>). The WCAG's goal is to provide a single shared standard for web content accessibility that meets the needs of individuals, organizations and governments internationally. Over time, WCAG has produced various versions of the [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/). WCAG 2.0 was published on 11 December 2008. WCAG 2.1 was published on 5 June 2018. WCAG 2.2 is scheduled to be published in 2021.
```

Comment:
```suggestion
The [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/) (<abbr>WCAG</abbr>) are web accessibility guidelines published by the [**Web Accessibility Initiative**](https://www.w3.org/WAI/about/) (<abbr>WAI</abbr>). 

The WCAG's goal is to provide a single shared standard for web content accessibility that meets the needs of individuals, organizations and governments internationally. Over time, WCAG has produced various versions of the [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/). WCAG 2.0 was published on 11 December 2008, WCAG 2.1 was published on 5 June 2018. WCAG 2.2 is a W3C working draft as of 5 June 2020.
```

Splitting this up to have a less dense intro. Also updating the bolding application to standardize.